api/src/main/java/com/epam/pipeline/controller/metadata/CategoricalAttributeController.java

```diff
@@ -47,14 +48,27 @@
 
     @PostMapping
     @ApiOperation(
-        value = "Update categorical attributes values.",
-        notes = "Update categorical attributes values",
+        value = "Create new categorical attribute.",
+        notes = "Create new categorical attribute.",
         produces = MediaType.APPLICATION_JSON_VALUE)
     @ApiResponses(
         value = {@ApiResponse(code = HTTP_STATUS_OK, message = API_STATUS_DESCRIPTION)
         })
-    public Result<Boolean> updateCategoricalAttributes(@RequestBody final List<CategoricalAttribute> dict) {
-        return Result.success(categoricalAttributeApiService.updateCategoricalAttributes(dict));
+    public Result<CategoricalAttribute> createCategoricalAttribute(@RequestBody final CategoricalAttribute attribute) {
```

Comment:
In order to minimize client changes I'd suggest to keep one endpoint for create/update. If `attribute` is passed with `id` call `update`, otherwise - `create`.

Reply:
Done